### ISSUE TITLE
CoreML: export failure: PyTorch convert function for op 'tensor_split' not implemented.

### ISSUE THREAD

**Issue by jaehyunshinML** [ultralytics/yolov5#7069]
Hi I tried to convert the custom model to cormel  with 

**python export.py --weights yolov5s.pt --include coreml** 

and 

got this error message.

**CoreML: export failure: PyTorch convert function for op 'tensor_split' not implemented.**

I installed all the package with requirements.txt in a new virtual condo environment.

but still has the same errors. 

do you may know possible solution for this? Thanks..

**Comment by dmitriy-kalashnikov** [comment]
Hello, I have the same issue

python3 export.py --weights yolov5l6.pt --include engine --device 0

`export: data=data/coco128.yaml, weights=['yolov5l6.pt'], imgsz=[640, 640], batch_size=1, device=0, half=False, inplace=False, train=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=12, verbose=False, workspace=4, nms=False, agnostic_nms=False, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['engine']
YOLOv5 🚀 v6.1-51-g9cd89b7 torch 1.11.0+cu113 CUDA:0 (NVIDIA GeForce RTX 3090, 24265MiB)

Fusing layers...
Model Summary: 476 layers, 76726332 parameters, 0 gradients

PyTorch: starting from yolov5l6.pt with output shape (1, 25500, 85) (147.4 MB)

ONNX: starting export with onnx 1.11.0...
WARNING: The shape inference of prim::Constant type is missing, so it may result in wrong shape inference for the exported graph. Please consider adding it in symbolic function.
WARNING: The shape inference of prim::Constant type is missing, so it may result in wrong shape inference for the exported graph. Please consider adding it in symbolic function.
WARNING: The shape inference of prim::Constant type is missing, so it may result in wrong shape inference for the exported graph. Please consider adding it in symbolic function.
ONNX: export failure: Exporting the operator tensor_split to ONNX opset version 13 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub.

TensorRT: starting export with TensorRT 8.4.0.6...

TensorRT: export failure: failed to export ONNX file: yolov5l6.onnx
`

**Comment by glenn-jocher** [comment]
@dmitriy-kalashnikov @jaehyunshinML good news 😃! Your original issue may now be fixed ✅ in PR #7074. To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!

**Comment by dmitriy-kalashnikov** [comment]
Thanks, it's currently fixed.
